README.md

```diff
@@ -1,6 +1,6 @@
-# GOV.UK Notify Node.js client
+# GOV.AU Notify Node.js client
```

Comment:
Don't we just want `Notify` across the board not `GOV.AU Notify`? At the top we can do something like:

```md
# Notify Node.js client

This is a Node.js client for notify.gov.au...
```

Plus a link in the GitHub repo URL field.

Reply:
I meant that in the GitHub repo there is a field for website address—we can put https://notify.gov.au in there.

Reply:
Also, I checked and we do want to use `Notify` instead of `GOV.AU Notify`. Anywhere at the top level that it's not obvious we are in the context of "notify.gov.au" we should label it "Notify.gov.au". If that's not clear, don't worry too much about it for now.